### PR TITLE
Fixes #33714 - add facts and reports in host details kebab

### PR DESCRIPTION
**Pull request by amirfefer**
Screen Shot 2021-10-18 at 21 09 59

### CODE REVIEW

File: webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js [hunk 1]
```diff
@@ -46,21 +59,31 @@ const ActionsBar = ({
...
+      {__('Reports')}
     </DropdownItem>,
+    <DropdownSeparator />,
     <DropdownItem
       icon={<UndoIcon />}
       href={`/hosts/${hostId}`}
       key="prev-version"
     >
-      {__('Previous version')}
+      {__('Previous UI')}
```
Comment (ekohl) [inline webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js:86]: I agree `version` is odd and `UI` is better but I'm not entirely sure about `Previous`. How do you feel about `Classic UI`? Probably not great either. Another suggestions would be `Old UI`.

**Review by ekohl — COMMENTED** [review]
Personally I like it in the current UI that they're not hidden behind a kebab menu but rather directly visible. I use them quite often.

For reports I would consider showing the last x (3?) eventful reports for the host similar to how audits are shown. IIRC that also has a link to all audits and for reports you could do the same thing. In the table I would expect a timestamp, source and statuses. Ideally this component could later be (re)used on the dashboard (Foreman's landing page) which also has a latest eventful reports, but then for all hosts (so that would need a hostname too).

For facts there's a less natural way because you don't have latest x facts. However, we do have host details and many of them are derived from facts. That could be a logical place for a Facts button/link.

**Comment by amirfefer** [comment]
Thanks, @ekohl!
I agree about the recent report card, created a quick draft PR https://github.com/theforeman/foreman/pull/8858, I wonder how would we show each report status.

File: webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js (hunk 1 above)
Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js:86]: Can we have `Old UI`?

Comment (ezr-ondrej) [reply webpack/assets/javascripts/react_app/components/HostDetails/ActionsBar/index.js:86]: I'll open a PR for that as it doesn't belong to this PR anyway.
RM tracking it: https://projects.theforeman.org/issues/33742

**Review by ezr-ondrej — APPROVED** [review]
Thanks @amirfefer ! :+1:

**Pull request merged**